### ISSUE TITLE
otr-ssh reports successful connection even though the connection fails

### ISSUE THREAD

**Issue by NeoTheThird** [TronFortyTwo/OnTheRoad#6]
Even if the ssh connection fails, `otr-ssh` reports a successful connection. Example for #5:

```
------------------------------------------
NOW YOU ARE INSIDE THE APPARMOR-FREE SHELL

Now you can run 'otr'.
You can go back to the normal shell with 'exit'
------------------------------------------

Permission denied (publickey).
phablet@ubuntu-phablet:~$
```

**Comment by TronFortyTwo** [comment]
Thanks for your report. I'll look into it asap.

**Comment by TronFortyTwo** [comment]
This has been fixed in master, will be published with next release (probably 2.0.0)

**Comment by TronFortyTwo** [comment]
2.0.0 released with the fix
